distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -130,6 +118,26 @@ public void testCswRegistryUpdate() throws Exception {
     }
 
     @Test
+    public void testCswRegistryUpdateFailure() throws Exception {
+        String id = createRegistryEntry("urn:uuid:2014ca7f59ac46f495e32b4a67a51280");
+
+        given().body(Library.getCswRegistryUpdate()
+                .replaceAll("urn:uuid:2014ca7f59ac46f495e32b4a67a51276",
+                        "urn:uuid:2014ca7f59ac46f495e32b4a67a51280")
+                .replace("Node Name", "New Node Name")
+                .replace("2016-01-26T17:16:34.996Z", "2015-02-26T17:16:34.996Z")
+                .replaceAll("someUUID", id))
+                .header("Content-Type", "text/xml")
+                .expect()
+                .log()
+                .all()
+                .statusCode(400)
+                .when()
+                .post(CSW_PATH.getUrl());
```

Comment:
Thank you, I modified testCswRegistryUpdate and testCswRegistryCreateDup to what is stated above. Also removed the .expect()